Source/WebGPU/WebGPU/RenderPipeline.mm

```diff
@@ -548,14 +548,15 @@ static WGPUBindGroupLayoutEntry createEntryFromStructMember(MTLStructMember *str
 }
 #endif // HAVE(METAL_BUFFER_BINDING_REFLECTION)
 
-BindGroupLayout* RenderPipeline::getBindGroupLayout(uint32_t groupIndex)
+Ref<BindGroupLayout> RenderPipeline::getBindGroupLayout(uint32_t groupIndex, bool& createdLayout)
```

Comment:
I think this is wrong. Getters should return +0 objects, not +1 objects. Ref indicates it's a +1 object. Instead, getters are supposed to return a raw (or weak) pointer (or C++ reference) to an internally-owned object.